app/components/invites/steps/SelectFeedsStep.module.css

```
@@ -1,11 +1,21 @@
-.options {
-  display: flex;
-  flex-direction: column;
-  gap: var(--spacing8);
+.step {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr auto;
+  grid-template-areas: "addFeedsStep addFeedsStep"
```

Stylelint (17.2.0)

[error] Expected shorthand property "grid-template" (declaration-block-no-redundant-longhand-properties)

(declaration-block-no-redundant-longhand-properties)